common/djangoapps/util/views.py

```diff
@@ -434,6 +440,12 @@ def build_error_response(status_code, field, err_msg):
 
     success = False
     context = get_feedback_form_context(request)
+
+    #Update the tag info with 'enterprise_learner' if the user belongs to an enterprise customer.
+    enterprise_learner_data = enterprise_api.get_enterprise_learner_data(site=request.site, user=request.user)
```

Comment:
@brittneyexline this codepath can be triggered by anonymous users as well, as the feedback form appears on the login and registration page.